distributed/distributed.yaml

```diff
@@ -142,6 +142,10 @@ distributed:
   nanny:
     preload: []             # Run custom modules with Nanny
     preload-argv: []        # See https://docs.dask.org/en/latest/setup/custom-startup.html
+    environ:
+      MALLOC_TRIM_THRESHOLD_: 65536
```

Comment:
```suggestion
      # Linux only; inconsequential on Windows, MacOS, or when using alternative memory managers.
      # See https://distributed.dask.org/en/latest/worker.html#memory-not-released-back-to-the-os
      MALLOC_TRIM_THRESHOLD_: 65536
```

Reply:
Historically we've avoided putting lots of comments in the main config file, relying instead on the description in the schema file (which gets put into docs).  I'm also OK being silent about this because it doesn't have any negative effect.

Reply:
What's the reasoning behind this value for `MALLOC_TRIM_THRESHOLD_`?